test/util/tile_cover.test.cpp

```diff
@@ -34,13 +34,11 @@ TEST(TileCover, Pitch) {
     Transform transform;
     transform.resize({ 512, 512 });
     // slightly offset center so that tile order is better defined
-    transform.setLatLng({ 0.1, -0.1 });
-    transform.setZoom(2);
-    transform.setAngle(5.0);
-    transform.setPitch(40.0 * M_PI / 180.0);
+
+    transform.jumpTo(CameraOptions().withCenter(LatLng { 0.1, -0.1, }).withZoom(2.0).withAngle(5.0).withPitch(40.0));
 
     EXPECT_EQ((std::vector<UnwrappedTileID>{
-        { 2, 1, 2 }, { 2, 1, 1 }, { 2, 2, 2 }, { 2, 2, 1 }, { 2, 3, 2 }
+        { 2, 1, 1 }, { 2, 2, 1 }, { 2, 1, 2 }, { 2, 2, 2 }
```

Comment:
The order of results returned by tile cover is important. It is supposed to radiate out from the center. Is there a reason that the order and number of tiles is changing?

Reply:
@asheemmamoowala yes - previously, the bearing (angle) was set in the transform object, which receives bearing in radians. The value, however, is `5.0`, which I assume it is supposed to indicate a value in degrees. The updated code fixes this typo and has the bearing value set via `CameraOptions`, which now properly receives degree values for both bearing and pitch.